Cartesian_kernel/include/CGAL/Cartesian/function_objects.h

```diff
@@ -3178,6 +3179,10 @@ namespace CartesianKernelFunctors {
     operator()( const Ray_3& r, const Point_3& p ) const
     { return CommonKernelFunctors::Construct_projected_point_3<K>()(r,p,K()); }
 
+    Point_3
```

Comment:
If we add that here, do we also have to add it in the Homogeneous kernel or in Kernel_23 ?
We also have to add it to the concept.

Reply:
In fact it seems that the code IS also in Kernel_23